### PR TITLE
Wenting0905 patch 3

### PR DESCRIPTION
**Pull request by Wenting0905**
PTAL @shenli

### CODE REVIEW

**Comment by shenli** [comment]
LGTM @siddontang PTAL

File: README.md [hunk 1]
```diff
@@ -12,17 +12,20 @@
...
-	+ [TiDB SQL Grammar](https://pingcap.github.io/sqlgram/)
+	+ [TiDB SQL Grammar](sql/README.md)
 	+ [Compatibility with MySQL](op-guide/mysql-compatibility.md)
 + [Frequently Used Tools](https://github.com/pingcap/tidb-tools)
 + [Frequently Asked Questions (FAQ)](./TiDB_FAQ.md)
 + [Troubleshooting](./trouble-shooting.md)
 + Use Cases
 + Advanced Usage
 	+ [Data migration from MySQL to TiDB](op-guide/migration.md)
+	+ [Loader instructions](loader.md)
```
Comment (siddontang) [inline README.md]: invalid reference for loader

Comment (siddontang) [inline README.md]: `./tool/loader.md`

**Comment by siddontang** [comment]
LGTM

**Pull request merged**